submissions/karelskiy/tiny-JS-world/index.js

```diff
@@ -0,0 +1,39 @@
+const population = [
+    {
+        species: 'dog',
+        name: 'Toby',
+        gender: 'male',
+        legs: 4,
+        hands: 0,
+        saying: 'woof-woof!'
+    },
+    {
+        species: 'cat',
+        name: 'Mrs. Velasquez',
+        gender: 'female',
+        legs: 4,
+        hands: 0,
+        saying: 'meow-meow!'
+    },
+    {
+        species: 'human',
+        name: 'Enrico',
+        gender: 'male',
+        legs: 2,
+        hands: 2,
+        saying: 'Holla'
+    },
+    {
+        species: 'human',
+        name: 'Angel',
+        gender: 'female',
+        legs: 2,
+        hands: 2,
+        saying: 'Ehi, buongiorno'
+    }
+    
+];
+
+population.forEach(el => {
+    print(`${el['species']}; ${el['name']}; ${el['gender']}; ${el['legs']}; ${el['hands']}; ${el['saying']}`);
+})
```

Comment:
Take a note of the red mark added by GitHub.
It is good to have the file ending with a newline for a number of reasons.
You do not have to check this and fix manually. Make your IDE doing this for you.
Find the relevant setting in your IDE and set is as appropriate.
In IntelliJ IDEA tools it is **Preferences > Editor > General > Ensure line feed at file end on Save**. Other IDEs offer similar.

Reply:
Ok, thanks for advice.